cmd/plan.go

```diff
@@ -146,8 +157,8 @@ func GetPlanCommand() *cobra.Command {
 	planCmd.Flags().StringVarP(&flags.srcpath, sourceFlag, "s", ".", "Specify source directory.")
 	planCmd.Flags().StringVarP(&flags.planfile, planFlag, "p", common.DefaultPlanFile, "Specify a file path to save plan to.")
 	planCmd.Flags().StringVarP(&flags.name, nameFlag, "n", common.DefaultProjectName, "Specify the project name.")
-	planCmd.Flags().StringVarP(&flags.customizationsPath, customizationsFlag, "c", common.DefaultCustomizationFolder, "Specify directory where customizations are stored.")
-	planCmd.Flags().StringSliceVarP(&flags.configs, configFlag, "f", []string{common.DefaultConfigFilePath}, "Specify config file locations.")
+	planCmd.Flags().StringVarP(&flags.customizationsPath, customizationsFlag, "c", "", "Specify directory where customizations are stored. By default we look for "+common.DefaultCustomizationDir)
```

Comment:
In addition to mentioning by default we look for the default dir, do we need to add "if present"? or is it obvious that if it is not present it will be ignored?

Reply:
I think the wording "look for" is fine because we do the same for the plan file.